src/env.cc

```diff
@@ -30,6 +34,11 @@ void Environment::Start(int argc,
   HandleScope handle_scope(isolate());
   Context::Scope context_scope(context());
 
+#if HAVE_INSPECTOR
+  ContextCreated(
+    new node::inspector::ContextInfo(context(), 1, "NodeJS Main Context"));
```

Comment:
Line continuations should have four spaces of indent.  (Ditto in other files, I won't point them out separately.)